src/eip7594/fk20.c

```diff
@@ -49,7 +65,57 @@ static void toeplitz_coeffs_stride(fr_t *out, const fr_t *in, size_t offset) {
 }
```

Comment:
This approach for filling the array fills more complex than it should be. Here are a few things we could improve:
- `out_start` is not a great var name, especially when the array is called `out`. Also what is `in_start`? What starts based on those variables? Maybe we should doc or rename.
- This function manages to use all the constants somehow: `CELLS_PER_BLOB`, `FIELD_ELEMENTS_PER_BLOB`, `CELLS_PER_EXT_BLOB`, `FIELD_ELEMENTS_PER_CELL`. It's really hard to read it and be confident that it actually fills the entire array.

If the bug is still here, is there a way to make this code a bit more readable?